package.json

```diff
@@ -123,7 +129,7 @@
         "ts-node": "7.0.1",
         "typescript": "3.3.1",
         "vscode": "1.1.33",
-        "vscode-azureextensiondev": "0.1.8"
+        "vscode-azureextensiondev": "0.2.1"
```

Comment:
Does this fix the security vulnerability with this package?

Reply:
The intent was not to fix the vulnerability. I'll need to investigate what exactly was the security vulnerability and will get back on this.

Reply:
This should solve the security vulnerability as this is the latest package: https://www.npmjs.com/package/vscode-azureextensiondev